### PR TITLE
New: Sohu.SHPlayer version 7.0.6.0

### PR DESCRIPTION
**Pull request by SpecterShell**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/26402)

### CODE REVIEW

File: manifests/s/Sohu/SHPlayer/7.0.6.0/Sohu.SHPlayer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+- vob
+- webm
+- wmv
+InstallModes:
+- interactive
+- silent
+Installers:
+- InstallerLocale: zh-CN
+  Architecture: x86
+  InstallerType: exe
```
Comment (KaranKad) [inline manifests/s/Sohu/SHPlayer/7.0.6.0/Sohu.SHPlayer.installer.yaml:42]:
```suggestion
  InstallerType: nullsoft
```

File: manifests/s/Sohu/SHPlayer/7.0.6.0/Sohu.SHPlayer.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  Scope: user
+  InstallerUrl: https://data.vod.itc.cn/foxd/gz?file=SohuNewPlayer_7.0.6.0_08_27_21_18_New7z_All.exe&new=/219/218/fSBHnvi8psbXi0pcaCXTXA.exe
+  InstallerSha256: C501F24FD6F04D2A1FFEFA03AA838DDE3D87016BF521874459B6ABE5CF3834DB
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /S
```
Comment (KaranKad) [inline manifests/s/Sohu/SHPlayer/7.0.6.0/Sohu.SHPlayer.installer.yaml:48]:
```suggestion
```

**Review by KaranKad — CHANGES_REQUESTED** [review]

File: manifests/s/Sohu/SHPlayer/7.0.6.0/Sohu.SHPlayer.installer.yaml (hunk 1 above)
Comment (SpecterShell) [reply manifests/s/Sohu/SHPlayer/7.0.6.0/Sohu.SHPlayer.installer.yaml:42]: Although the installer uses switches similar to NSIS, many detectors including 7-zip cannot recognize it as a Nullsoft Installer. So I change to use custom switches.🤔

**Review by KaranKad — COMMENTED** [review]
True, it seems like its different type of installer so going with custom switch would be better.

**Review by KaranKad — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**